pkg/sdk/object_types.go

```diff
@@ -68,6 +68,10 @@ const (
 	ObjectTypeHybridTable          ObjectType = "HYBRID TABLE"
 	ObjectTypeImageRepository      ObjectType = "IMAGE REPOSITORY"
 	ObjectTypeProjectionPolicy     ObjectType = "PROJECTION POLICY"
+	ObjectTypeDataMetricFunction   ObjectType = "DATA METRIC FUNCTION"
```

Comment:
Have you checked that they does not have to be added to any other resources? Please check it.